catalog/gke/cluster/cluster.yaml

```diff
@@ -88,12 +100,11 @@ spec:
   # Use a dedicated subnet by default, to increase isolation and allow for
   # cluster-specific firewalls.
   subnetworkRef:
-    name: platform-project-id-example-us-east4 # kpt-set: ${platform-project-id}-${cluster-name}
-    namespace: networking # kpt-set: ${networking-namespace}
+    external: projects/network-project-id/regions/us-west4/subnetworks/default # kpt-set: projects/${network-project-id}/regions/${location}/subnetworks/${subnet-name}
```

Comment:
Couldn't location become a zone for a zonal cluster which would then make this selflink invalid?

Reply:
changed to include network-ref and subnet-ref instead of interpolating the string